src/language/records.md

```diff
@@ -64,21 +67,25 @@ the [record's type definition](#record-types), or its _shape_.
 Two records with named fields with
 different names have different types:
 
+<?code-excerpt "language/test/records_test.dart (record-type-mismatched-names)"?>
 ```dart
 ({int a, int b}) recordAB = (a: 1, b: 2);
 ({int x, int y}) recordXY = (x: 3, y: 4);
 
-recordAB = recordXY; // Compile error! These records don't have the same type.
+// Compile error! These records don't have the same type.
+// recordAB = recordXY;
```

Comment:
I could do this as separate follow-up, but I wanted to note:

Consider removing the comments before this, so it is actually a compile error.

You can use the code-excerpt replace functionality to remove them. 

Then you can add `{:.fails-sa}` before the code excerpt so it's clear it isn't a valid code snippet:

<img width="334" alt="Example of fails-sa" src="https://github.com/dart-lang/site-www/assets/18372958/667146e6-49d0-40a2-a290-a17443f18f0e">

Reply:
How exactly to hold the `replace` command is eluding me, so I will take you up on the offer to do this in a second pass. Please tag me when you do, though - I'd love to see the right incantation.

Reply:
Sounds good. I'll open that tonight or tomorrow and make sure to tag you :)